_data/projects/xubuntu-18.04-dev-box.yml

```diff
@@ -0,0 +1,13 @@
+name: xubuntu-18.04-dev-box
+desc: Throwaway development environment in Xubuntu using Vagrant, Virtualbox, and Ansible
+site: https://github.com/SteveParson/xubuntu-18.04-dev-box
+tags:
+- ansible
+- bash
+- java
+- python
+- vagrant
+- virtualbox
+upforgrabs:
+  name: up-for-grabs
+link: https://github.com/SteveParson/xubuntu-18.04-dev-box/labels/up-for-grabs
```

Comment:
The indenting on this line is off - it should be on the same indent as `name`

Reply:
Thanks, I've corrected the error